config/muxcable.py

```diff
@@ -254,7 +254,7 @@ def lookup_statedb_and_update_configdb(db, per_npu_statedb, config_db, port, sta
     if str(state_cfg_val) == str(configdb_state):
         port_status_dict[port_name] = 'OK'
     else:
-        if cable_type is not None and soc_ipv4_value is not None:
+        if cable_type is not None or soc_ipv4_value is not None:
```

Comment:
Are we assuming `cable_type` field only presents for `active-active` ports? It doesn't harm to check `if cable_type == "active-active"`?

Reply:
so basically, for "active-standby" type cable there is no soc_ipv4 but we still need to write the value of "cable_type" back